app/controllers/api/v1/flagged_applications_controller.rb

```diff
@@ -21,7 +47,27 @@ def flagged_application_params
           )
   end
 
-  def service
+  def flagged_applications_get_params
+    params.require(:type)
+  end
+
+  def flagged_applications_record_set_params
+    params.require(:id)
+  end
+
+  def pending_data
+    flagged_record_set_get_service.pending_review_record_sets
+    # fields = flagged_record_set_get_service.pending_review_index_fields
+    # [pending_review_record_sets, fields]
+  end
+
+  def duplicated_data
+    flagged_record_set_get_service.marked_duplicate_record_sets
+    # fields = flagged_record_set_get_service.marked_duplicate_index_fields
+    # [marked_duplicate_record_sets, fields]
```

Comment:
**question:** Do we need these commented out lines? You are already returning the record set and it doesnt look like fields were ever used in FlaggedApplicationsIndexPage or FlaggedApplicationsShowPage

Reply:
These commented out lines were in use (but not used by React) before I moved data fetching to the Rails API. These comments can be removed, since we can reference the blame if we ever finding ourselves needing `fields` again.